### ISSUE TITLE
Crash when changing node type

### ISSUE THREAD

**Issue by ilesinge** [emprcl/signls#1]
Steps to reproduce:

- Add an Euclid node
- Edit the node
- Select the 6th param (stp)
- Type "3" to change the node type

```
Caught panic:

runtime error: index out of range [5] with length 5

Restoring terminal...

goroutine 1 [running]:
runtime/debug.Stack()
	/opt/hostedtoolcache/go/1.23.2/x64/src/runtime/debug/stack.go:26 +0x5e
runtime/debug.PrintStack()
	/opt/hostedtoolcache/go/1.23.2/x64/src/runtime/debug/stack.go:18 +0x13
github.com/charmbracelet/bubbletea.(*Program).Run.func1()
	/home/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v1.1.0/tea.go:519 +0x8b
panic({0x5a76a0?, 0xc0003160a8?})
	/opt/hostedtoolcache/go/1.23.2/x64/src/runtime/panic.go:785 +0x132
signls/ui.mainModel.renderNode({0xc000074440, 0xc000119180, {0x0, 0x0, 0x27, 0x20}, {{{0xc000016390, 0x1, 0x1}, {{...}, ...}, ...}, ...}, ...}, ...)
	/home/runner/work/signls/signls/ui/node.go:51 +0x1050
signls/ui.mainModel.renderGrid({0xc000074440, 0xc000119180, {0x0, 0x0, 0x27, 0x20}, {{{0xc000016390, 0x1, 0x1}, {{...}, ...}, ...}, ...}, ...})
	/home/runner/work/signls/signls/ui/ui.go:372 +0x308
signls/ui.mainModel.View({0xc000074440, 0xc000119180, {0x0, 0x0, 0x27, 0x20}, {{{0xc000016390, 0x1, 0x1}, {{...}, ...}, ...}, ...}, ...})
	/home/runner/work/signls/signls/ui/ui.go:318 +0x514
github.com/charmbracelet/bubbletea.(*Program).eventLoop(0xc00011c240, {0x60bf50?, 0xc0001a0008?}, 0xc0001021c0)
	/home/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v1.1.0/tea.go:454 +0x84d
github.com/charmbracelet/bubbletea.(*Program).Run(0xc00011c240)
	/home/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v1.1.0/tea.go:593 +0xa32
main.main()
	/home/runner/work/signls/signls/main.go:55 +0x556
```

**Comment by xaviergodart** [comment]
Thanks!
It should be fixed in [v0.1.5](https://github.com/emprcl/signls/releases/tag/v0.1.5).
Could you test it?

**Comment by ilesinge** [comment]
It's fixed :)
